app/jobs/regular/user_email.rb

```diff
@@ -108,8 +108,12 @@ def message_for_email(user, post, type, notification, args = nil)
         return skip_message(SkippedEmailLog.reason_types[:user_email_anonymous_user])
       end
 
-      if user.suspended? && !%w[user_private_message account_suspended].include?(type.to_s)
-        return skip_message(SkippedEmailLog.reason_types[:user_email_user_suspended_not_pm])
+      if user.suspended?
+        if !type.to_s.in?(%w[user_private_message account_suspended])
```

Comment:
Not related to your PR but we seem to be doing `type.to_s` a lot in this file. Would it make sense to do it once at the beginning of the method? 🤔

Reply:
Yes, we can probably improve some areas in this file :slightly_smiling_face: I can create a small refactor PR and see where it takes us :grin: